changelogs/fragments/module_utils_s3-unit-testing.yml

```
@@ -0,0 +1,3 @@
+---
+trivial:
```

```suggestion
minor_changes:
```

`Trivial` would be something that doesn't show up in the changelog, which makes sense for things like docs/test changes.  There is a risk that this refactor breaks something...